conf/mds.conf

```diff
@@ -252,3 +252,20 @@ mds.throttle.bpsPerGBInMB=0.3
 # - if filename is /dir1/file, then poolset1 is select
 # - if filename is /dir1/sub/file, then sub is select
 mds.poolset.rules=
+
+#### auth options ####
+#
+# auth server option
+auth.server.key=5ef3e1a5c95736e0  # key is 128bit (16 bytes)
```

Comment:
Is this secret key stored in the configuration file in plain text?

Reply:
default is null, you shuold add key in topology.yaml when deploy cluster and that key will pass to conf.